spec/nanaimo/reader_spec.rb

```diff
@@ -139,7 +139,13 @@ module Nanaimo
         let(:data) { '<12 3>' }
 
         it 'raises an informative error' do
-          expect { subject }.to raise_error(Reader::ParseError, 'Data has an uneven number of hex digits')
+          expect { subject }.to raise_error(Reader::ParseError, <<-E)
+[!] Data has an uneven number of hex digits
+ #  -------------------------------------------
+ >  {key = <12 3>}
```

Comment:
Might be nice if this was "#{line_number}> #{line_content}" i.e `1> {key = <12 3>}`